src/main/java/com/fundynamic/d2tm/game/behaviors/SelectableImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.fundynamic.d2tm.game.behaviors;
+
+import com.fundynamic.d2tm.game.map.MapEntity;
+import org.newdawn.slick.Color;
+import org.newdawn.slick.Graphics;
+
+public class SelectableImpl {
```

Comment:
Since we have a `Selectable` interface, I'd expect `SelectableImpl` to extend from it (based on the normal naming convention). So maybe rename this to `SelectionOutline` or something like that?